source/mir/ion/examples.d

```diff
@@ -370,14 +370,36 @@ version(mir_ion_test) unittest
     import mir.ser.json;
     import mir.deser.json;
 
+    static struct CustomType
+    {
+        int a;
+
+        // can be part of a type as well (only omits on property keys)
+        bool serdeIgnoreOut() const
+        {
+            return a < 0;
+        }
+
+        // and use custom serializer to serialize as int
+        void serialize(S)(ref S serializer) const
+        {
+            import mir.ser : serializeValue;
+
+            serializeValue(serializer, a);
```

Comment:
nit: please use `serializer.putValue` instead of `serializeValue` here

Reply:
why putValue? Wouldn't that skip custom serialize handlers if `a` was some custom type?